### ISSUE TITLE
"Local host name unknown" causing container to not start 

### ISSUE THREAD

**Issue by billglick** [ctindel/ubiquiti-docker#53]
After reinstalling this fresh, I'm now seeing the docker image continuously reboot. The logs show the following message every time it restarts:

`kernel: [ 5915.816251] aufs au_opts_verify:1571:dockerd[27535]: dirperm1 breaks the protection by the permission bits on the lower branch`

**Comment by billglick** [comment]
This is with the following environment:

- Synology RS812RP+
- DSM 6.1.6-15266
- Docker 17.05.0-ce, build 9f07f0e-synology
- Container Image ctindel/unifi-video-controller:3.9.4

**Comment by billglick** [comment]
I figured out my issue. It didn't really have anything to do with the error I was seeing above.

When I looked at the terminal output of my container failing to start, I saw a message about "Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: unifi-video: unifi-video: Name or service not known" and then "Service exit with a return value of 1".
<img width="651" alt="unifi-docker-error-screenshot 2018-04-15 17 37 13" src="https://user-images.githubusercontent.com/910479/38784401-f01aaad0-40d6-11e8-81d8-8d9bf79581f9.png">

I was able to fix the problem by adding "unifi-video" as an alias for localhost in my /etc/hosts file. Perhaps there is a better way to handle this.
